### PR TITLE
Platinum Skill: skill MC_CARTDECORATE

### PR DESCRIPTION
**Pull request by mazvi**
Added MC_CARTDECORATE for Merchant Class

* **Addressed Issue(s)**: 

* **Server Mode**: 

* **Description of Pull Request**:

### CODE REVIEW

**Comment by anacondaqq** [comment]
add condition check before adding the skill:
```c
if(PACKETVER >= 20150826)
{
 .... your code
}
```

why? Because the skill can work with 20150826+ clients.

File: npc/custom/platinum_skills.txt [hunk 1]
```diff
@@ -55,6 +56,7 @@ prontera,128,200,6	script	Platinum Skill NPC	94,{
 		skill "MC_CARTREVOLUTION",1,SKILL_PERM;
 		skill "MC_CHANGECART",1,SKILL_PERM;
 		skill "MC_LOUD",1,SKILL_PERM;
+		skill "MC_CARTDECORATE",1,SKILL_PERM;
```
Comment (anacondaqq) [inline npc/custom/platinum_skills.txt]:
```
if(PACKETVER >= 20150826)
	skill "MC_CARTDECORATE",1,SKILL_PERM;
```

Comment (mazvi) [reply npc/custom/platinum_skills.txt]: thanks i was added if(PACKETVER >= 20150826) above skill "MC_CARTDECORATE",1,SKILL_PERM;

File: npc/custom/platinum_skills.txt [hunk 2]
```diff
@@ -55,6 +57,8 @@ prontera,128,200,6	script	Platinum Skill NPC	94,{
 		skill "MC_CARTREVOLUTION",1,SKILL_PERM;
 		skill "MC_CHANGECART",1,SKILL_PERM;
 		skill "MC_LOUD",1,SKILL_PERM;
+		if(PACKETVER >= 20150826)
+		skill "MC_CARTDECORATE",1,SKILL_PERM;
```
Comment (secretdataz) [inline npc/custom/platinum_skills.txt]: Indent this line.

Comment (mazvi) [reply npc/custom/platinum_skills.txt]: thanks line has been indent

**Comment by mazvi** [comment]
Travis tony stark ended with log The command "sudo apt-get install gcc-5 -y" failed and exited with 100 during .

**Pull request merged**